src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -3680,6 +3680,8 @@ components:
           description: Project id of the snapshot data project
         accessInformation:
           $ref: '#/components/schemas/AccessInfoModel'
+        creationInformation:
```

Comment:
I'm not in love with `creationInformation`. `snapshotSpecification` or just `specification` seems more in-line with what we call this type of info for other things in TDR, like assets.

Reply:
Not sure I agree, AssetSpecification doesn't look similar to this, and I don't like overloading generic, nonspecific terms like specification